internal/manifests/manifestutils/tls.go

```diff
@@ -117,3 +125,13 @@ func findContainerIndex(pod *corev1.PodSpec, containerName string) (int, error)
 
 	return -1, fmt.Errorf("cannot find container %s", containerName)
 }
+
+func findVolumeIndex(pod *corev1.PodSpec, volumeName string) (int, error) {
```

Comment:
what is the point of returning error if it is always ignored? It seems like the function could be renamed to `containsVolume(pod, volumeName) bool`

Reply:
Done